plugin/keyring/common/keyring_key.h

```diff
@@ -1,10 +1,4 @@
-<<<<<<< HEAD
 /* Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
-||||||| e5d189ecb94
-/* Copyright (c) 2016, 2017, Oracle and/or its affiliates. All rights reserved.
-=======
-/* Copyright (c) 2016, 2021, Oracle and/or its affiliates.
```

Comment:
Looks like this line should have stayed here

Reply:
Good call. I will fix it